drivers/serial.c

```diff
@@ -221,6 +221,19 @@ int serial_write(io_port_t port, const char *buf, size_t len) {
     return rc;
 }
 
+#define NUM_PLUS 3
+static inline void uart_reboot(char c) {
+    static volatile uint8_t break_cmd = NUM_PLUS;
```

Comment:
Why `volatile`?

Reply:
Ugh, copy and paste error from times I wanted to update this variable from the interrupt handler.

Reply:
But even then, and IRQ cannot interrupt itself, so this would only be an issue when the variable is shared, e.g. between the interrupt handler and some "bottom halves" code, which is not the case here.